src/sentry/snuba/discover.py

```diff
@@ -394,6 +394,18 @@ def create_result_key(
     result_row: SnubaRow, fields: list[str], issues: Mapping[int, str | None]
 ) -> str:
     """Create the string key to be used in the top events result dictionary"""
+    values = create_result_values(result_row, fields, issues)
+    result = ",".join(values)
+    # If the result would be identical to the other key, include the field name
+    # only need the first field since this would only happen with a single field
+    if result == OTHER_KEY:
+        result = f"{result} ({fields[0]})"
+    return result
+
+
+def create_result_values(
+    result_row: SnubaRow, fields: list[str], issues: Mapping[int, str | None]
+) -> list[str]:
```

Comment:
no functional change, pulling out the functionality so it can be used to construct the timeseries `groupby`